File: 2-semestre/uri-online-judge/javascript/1134.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let fim = 0, alc = 0, gas = 0, die = 0;

do {
    let painel = parseInt(lines.shift());

    if (painel === 1) alc++;
    else if (painel == 2) gas++;
    else if (painel == 3) die++;
    else if (painel == 4) fim = 4;
} while (fim != 4);

console.log(`MUITO OBRIGADO\nAlcool: ${alc}\nGasolina: ${gas}\nDiesel: ${die}`);

File: 2-semestre/uri-online-judge/c/1134.c
#include <stdio.h>
 
int main() {
    int fim = 0, painel, alc = 0, gas = 0, die = 0;

    do {
        scanf("%d", &painel);

        if (painel == 1) alc++;
        else if (painel == 2) gas++;
        else if (painel == 3) die++;
        else if (painel == 4) fim = 4;
    } while (fim != 4);

    printf("MUITO OBRIGADO\nAlcool: %d\nGasolina: %d\nDiesel: %d\n", alc, gas, die);

    return 0;
}
